### ISSUE TITLE
Export project as excel files

### ISSUE THREAD

**Issue by Felixmil** [esqLABS/esqlabsR#716]

**Comment by PavelBal** [comment]
You mean - json snapshot to our internal excel structure?

**Comment by Felixmil** [comment]
This is related to the new project based workflow. The goal is to be able to change project configuration in:
- R (modifying the project object)
- In Excel (in configuration files like we do today)
- In Json files

**Comment by Felixmil** [comment]
No, I need to rework the project approach to make it work with the new json snapshot
